tests/data/stack_aggregator_empty_resolved.json

```diff
@@ -0,0 +1,25 @@
+{
+    "external_request_id": "some_external_request_id",
+    "result": [{
```

Comment:
@sara-02 can we have tests for multiple results, one with `resolved` packages and another one without. That'll ensure that the code is working as expected.

Reply:
@samuzzal-choudhury AFAIK there is already one test for the normal one. This one
https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/tests/test_recommender.py#L77 and it picks the data from this file
https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/tests/data/stack_aggregator_execute_input.json

Reply:
Or do you want to test with same json having 2 results, one with resolved and one without?